internal/pkg/cli/pipeline_update.go

```diff
@@ -197,6 +210,36 @@ func (o *updatePipelineOpts) deployPipeline(in *deploy.CreatePipelineInput) erro
 	return nil
 }
 
+func (o *updatePipelineOpts) ConnectionName() (string, error) {
```

Comment:
nit: usually we define public methods when we have to.
```suggestion
func (o *updatePipelineOpts) connectionName() (string, error) {
```